src/App/ComponentsDocumentation/components/Slab/index.js

```diff
@@ -1,210 +1,66 @@
 import React, { Component } from "react";
 
 import { ComponentPreview, DocContainer } from "@docutils";
+
 import CodeTags from "@components/CodeTags";
 
+import { overviewSlabs } from "./constants";
 import { validation } from "@src/scripts/main";
```

Comment:
This validation thing is old I think.. Remove it all, also the useEffect longer down in the file.